draft-bbs-signatures.md

```diff
@@ -1382,6 +1384,14 @@ Along with the SK value as defined in (#key-pair) as inputs into the Sign operat
 
 This document does not make any requests of IANA.
 
+# Acknowledgements
+
+The authors would like to acknowledge the significant amount of academic work that preceeded the development of this document. In particular the original work of [@SGS] which was subsequently developed in [@CDL16].
```

Comment:
```suggestion
The authors would like to acknowledge the significant amount of academic work that preceeded the development of this document. In particular the original work of [@BBS04] which was subsequently developed in [@ASM06] and in [@CDL16]. This last academic work is the one mostly used by this document.
```
As far as i know of, It's more customary to cite using the authors initials with the year of publishing (at least in this format).

Also added one more paper.